src/main/java/duke/ui/Parser.java

```diff
@@ -0,0 +1,48 @@
+package duke.ui;
+
+import duke.commands.Commands;
+import duke.tasks.TaskManager;
+
+import java.util.Scanner;
+
+public class Parser {
+
+    static final Scanner sc = new Scanner(System.in);
+
+    private static void process(String inputMessage) {
+        if (inputMessage.trim().equals(Commands.list.toString())) {
+            Display.printStylised(TaskManager.list());
+        } else if (inputMessage.startsWith(Commands.mark.toString())) {
+            Display.printStylised(TaskManager.mark(Integer.parseInt(inputMessage.substring(Commands.mark.toString().length() + 1))));
```

Comment:
This is quite the eye-sore (on git PR view at least) consider storing some of that as a local variable